website/content/docs/connect/gateways/api-gateway/configuration/httprouteauthfilter.mdx

```diff
@@ -0,0 +1,139 @@
+- [`spec`](#spec): map | required
+   - [`jwt`](#spec-jwt): map | required 
+      - [`providers`](#spec-providers): list | required 
+         - [`name`](#spec-providers): string | required 
+         - [`verifyClaims`](#spec-providers): map | required 
+            - [`path`](#spec-providers): list of strings | required 
+            - [`value`](#spec-providers): string | required 
+
+## Complete configuration
+
+When every field is defined, an HTTP route auth filter has the following form: 
+
+## Specification
+
+This section provides details about the fields you can configure in the HTTP route auth filter resource.
+
+### `apiVersion`
+
+Specifies the version of the Consul API for integrating with Kubernetes. The value must be `consul.hashicorp.com/v1alpha1`.
+
+#### Values
+
+- Default: None
+- This field is required.
+- String value that must be set to `consul.hashicorp.com/v1alpha1`.
+
+### `kind`
+
+Specifies the type of configuration entry to implement. Must be set to `HTTPRouteAuthFilter`.
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: String value that must be set to `HTTPRouteAuthFilter`.
+
+### `metadata`
+
+Map that contains an arbitrary name for the resource and the namespace it applies to.
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `metadata.name`
+
+Specifies a name for the resource. The name is metadata that you can use to reference the resource when performing Consul operations, such as applying the resource to a specific cluster. 
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: String
+
+### `metadata.namespace` 
+
+Specifies the namespace that the configuration applies to. Refer to [Namespaces](/consul/docs/enterprise/namespaces) for more information.
+
+#### Values
+
+- Default: `default`
+- Data type: String
+
+### `spec`
+
+Map that contains the details about the HTTP route auth filter. The `apiVersion`, `kind`, and `metadata` fields are siblings of the `spec` field. All other configurations are children.
+
+#### Values
+
+- Default: None
+- This field is required.
+- Data type: Map
+
+
+### `spec.jwt`
+
+Map that contains JWT verification configurations to apply to listeners when the filter is attached to the route. Route-specific settings have precedence over the `default` configurations, but not `override` JWT configurations, defined in the [`GatewayPolicy`](/consul/docs/connect/gateways/api-gateway/configuration/gatewaypolicy) resource.
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `spec.jwt.providers`
+
+Specifies a list of JWT provider configurations to apply to listeners when the filter is attached to the route. A provider configuration contains the name of the provider and claims. The route-specific settings have precedence over the `default` configurations, but not `override` configurations, that are defined in the [`GatewayPolicy`](/consul/docs/connect/gateways/api-gateway/configuration/gatewaypolicy) resource.
```

Comment:
similar idea here, should we specify that we need an existing JWTProvider config entry to reference? (you can still create these without it existing but the jwt configuration won't work)